crates/ruff_linter/resources/test/fixtures/pyflakes/F401_26__all_empty/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""__init__.py with empty __all__
+"""
+
+
+from . import unused # F401: add to __all__
+
+
+from . import renamed as bees # F401: add to __all__
+
+
+__all__ = []
+
```

Comment:
Can you remove the trailing newlines here for consistency with the other fixtures? Otherwise, editors that automatically strip trailing newlines will lead to unnecessary edits here in the future. (You can run `ruff format` over these if it's easier.)

Reply:
Yes, sorry, I forgot to `ruff format` them